redis/typing.py

```diff
@@ -32,7 +32,14 @@
 PatternT = _StringLikeT  # Patterns matched against keys, fields etc
 FieldT = EncodableT  # Fields within hash tables, streams and geo commands
 KeysT = Union[KeyT, Iterable[KeyT]]
-ResponseT = Union[Awaitable[Any], Any]
+OldResponseT = Union[Awaitable[Any], Any]  # Deprecated
+AnyResponseT = TypeVar("AnyResponseT", bound=Any)
+ResponseT = Union[AnyResponseT, Awaitable[AnyResponseT]]
+OKT = Literal[True]
+ArrayResponseT = List
```

Comment:
Two aspects here.

First, `List` is itself generic, wouldn't it be cleaner to keep that into account? I guess in most places it would end up being `ArrayResponseT[Any]`, but still it would feel more correct.

Second, why define aliases to single types? You can just use `List` and it would be easier on the reader, I think.